smoke_tests/sqs_send_message_batch_test.go

```diff
@@ -318,27 +320,27 @@ func TestSendMessageBatchV1_Json_Success(t *testing.T) {
 			attr2.Name = k
 			attr2.Value = &models.ResultMessageAttributeValue{
 				DataType:    *attr.DataType,
-				StringValue: *attr.StringValue,
+				StringValue: attr.StringValue,
 			}
 		} else if k == numberAttribute {
 			attr3.Name = k
 			attr3.Value = &models.ResultMessageAttributeValue{
 				DataType:    *attr.DataType,
-				StringValue: *attr.StringValue,
+				StringValue: attr.StringValue,
 			}
 		}
 	}
 	assert.Equal(t, binaryAttribute, attr1.Name)
 	assert.Equal(t, binaryType, attr1.Value.DataType)
-	assert.Equal(t, "YmFzZTY0LWVuY29kZWQtdmFsdWU=", attr1.Value.BinaryValue) // base64 encoded value
+	assert.Equal(t, binaryValueString, attr1.Value.BinaryValue)
```

Comment:
Through the AWS SDK we should receive the raw string instead of the base 64 encoded string.